### PR TITLE
[ENH] BEP Update

### PR DESCRIPTION
**Pull request by franklin-feingold**
This provides an update for each of our BEP projects. Included is a status summary and potential blocking point(s). Here is the [rendering](https://874-150465237-gh.circle-artifacts.com/0/home/circleci/project/site/06-extensions.html).

@bids-standard/bep_leads - if you may please review your BEP summary and provide suggestions or enhancements if needed. Thank you!

One change is bringing back BEP012, 013, 016 into the extensions table because they were split from the common derivatives (https://github.com/bids-standard/bids-specification/issues/254)

### CODE REVIEW

**Comment by DejanDraschkow** [comment]
looks good for bep020

File: src/06-extensions.md [hunk 1]
```diff
@@ -28,27 +29,30 @@ which are community-driven processes.
...
+| [BEP013](https://docs.google.com/document/d/1qBNQimDx6CuvHjbDvuFyBIrf2WRFUOJ-u50canWjjaw) | The resting state fMRI derivatives                                                                         | Steven Giavasis                                                            |                                                                                                                                                                                                                                           | None.                                                                        |
+| [BEP014](https://docs.google.com/document/d/11gCzXOPUbYyuQx8fErtMO9tnOKC3kTWiL9axWkkILNE) | The affine transformations and nonlinear field warps                                                       | Oscar Esteban                                                              | This BEP is being refactored and will be released to the community for feedback.                                                                                                                                                          | Refactoring.                                                                 |
```
Comment (franklin-feingold) [inline src/06-extensions.md]: @oesteban - here is the line for providing an update on the BIDS Transforms

Comment (oesteban) [inline src/06-extensions.md]:
```suggestion
| [BEP014](https://docs.google.com/document/d/11gCzXOPUbYyuQx8fErtMO9tnOKC3kTWiL9axWkkILNE) | The affine transformations and nonlinear field warps                                                       | Oscar Esteban                                                              | A new file format (X5) to store spatial transforms is being created, and a draft proposal has been discussed under the scope of this BEP. A software prototype demonstrating X5 is currently under development.                           | In progress.                                                                 |
```

**Comment by effigies** [comment]
This looks like it's reverted almost all of its changes...

**Comment by franklin-feingold** [comment]
darn! will revert back to the edited version

**Comment by sappelhoff** [comment]
> This looks like it's reverted almost all of its changes...

thanks for catching that Chris! :grimacing:

**Comment by franklin-feingold** [comment]
reverted BEP018 back to the active list. Prepared this PR for a patch release. Can be updated when BEP018 is merged in for a minor release.

pinging @effigies @sappelhoff for review - here is [current rendering](https://876-150465237-gh.circle-artifacts.com/0/home/circleci/project/site/06-extensions.html)

File: src/06-extensions.md [hunk 2]
```diff
@@ -28,27 +29,30 @@ which are community-driven processes.
...
+| [BEP013](https://docs.google.com/document/d/1qBNQimDx6CuvHjbDvuFyBIrf2WRFUOJ-u50canWjjaw) | The resting state fMRI derivatives                                                                         | Steven Giavasis                                                            |                                                                                                                                                                                                                                           | None.                                                                                  |
```
Comment (effigies) [inline src/06-extensions.md]: This one was merged into BEP012.

Comment (franklin-feingold) [reply src/06-extensions.md]: I added a note of this for this update. In future updates it can be noted below the merged BEPs (e.g. BEP 6, 7, 8, 10)

**Comment by sappelhoff** [comment]
looks fine to me

**Comment by sappelhoff** [comment]
@franklin-feingold are you ready to have this one merged?

File: src/06-extensions.md [hunk 3]
```diff
@@ -28,27 +29,30 @@ which are community-driven processes.
...
+| [BEP001](https://docs.google.com/document/d/1QwfHyBzOyFWOLO4u_kkojLpUhW0-4_M7Ubafu9Gf4Gg) | Structural acquisitions that include multiple contrasts (multi echo, flip angle, inversion time) sequences | Gilles de Hollander                                                        | The draft BEP is nearly ready to be submitted as a PR.                                                                                                                                                                                    | A few items need to be revised and a final decision made.                              |
```
Comment (KirstieJane) [inline src/06-extensions.md]:
```suggestion
| [BEP001](https://github.com/bids-standard/bep001) | Structural acquisitions that include multiple contrasts (multi echo, flip angle, inversion time) sequences | Gilles de Hollander and Kirstie Whitaker                                                       | The draft BEP is nearly ready to be submitted as a PR.                                                                                                                                                                                    | A few items need to be revised and a final decision made.                              |
```

**Comment by franklin-feingold** [comment]
should be ready to merge - will commit a fix for travis

**Comment by franklin-feingold** [comment]
cc @sappelhoff @effigies should be ready to merge

**Comment by sappelhoff** [comment]
Thanks @franklin-feingold

**Pull request merged**